code/ship/ship.cpp

```diff
@@ -1902,6 +1916,12 @@ static void parse_ship(const char *filename, bool replace)
 		first_time = true;
 
 		strcpy_s(sip->name, fname);
+
+		// if this name has a hash, create a default display name
+		if (get_pointer_to_first_hash_symbol(sip->name)) {
```

Comment:
If I read this right, this could overwrite a previous display name for a `+nocreate` ship entry. Maybe only do this for `create_if_not_found`?

Reply:
Oh, true.  I'll fix that

Reply:
This should not overwrite any previous display name.  Look at the enclosing `if()` block; this is only run for brand new entries.

Reply:
Oh, yes. I misread that.